enterprise/lms_api.py

```diff
@@ -158,6 +221,48 @@ def get_remote_id(self, identity_provider, username):
         return None
 
 
+class GradesApiClient(JwtLmsApiClient):
+    """
+    Object builds an API client to make calls to the LMS Grades API.
+
+    Note that this API client requires a JWT token, and so it keeps its token alive.
+    """
+
+    API_BASE_URL = settings.LMS_ROOT_URL + '/api/grades/v0/'
+    APPEND_SLASH = True
+
+    @JwtLmsApiClient.refresh_token
+    def get_course_grade(self, course_id, username):
+        """
+        Retrieve the grade for the given username for the given course_id.
+
+        Args:
+        * ``course_id`` (str): The string value of the course's unique identifier
+        * ``username`` (str): The username ID identifying the user for which to retrieve the grade.
+
+        Returns:
+
+        None if no grade found, or a dict containing:
+
+        Raises:
+
+        HttpNotFoundError if no grade returned for the given user.
+
+        * ``username``: A string representation of a user's username passed in the request.
+        * ``course_key``: A string representation of a Course ID.
+        * ``passed``: Boolean representing whether the course has been passed according the course's grading policy.
+        * ``percent``: A float representing the overall grade for the course
+        * ``letter_grade``: A letter grade as defined in grading_policy (e.g. 'A' 'B' 'C' for 6.002x) or None
+
+        """
+        results = self.client.course_grade(course_id).users().get(username=username)
+        for row in results:
+            if row.get('username') == username:
```

Comment:
No, I'm just being overly thorough probably..  Since the Grades API returns a list of results, it looks like they might one day return more than one, e.g. 'fetch all grades for this course'.

But currently, the Grades API only returns one set of grades for the requested `username`, and throws a 404 if you're not allowed to see them.